src/commands.ts

```diff
@@ -1,28 +1,63 @@
-import { TransferConfig } from 'zkbob-client-js';
-import { TransferRequest, TreeState } from 'zkbob-client-js/lib/client';
-import { ProverMode } from 'zkbob-client-js/lib/config';
+import { EphemeralAddress, HistoryRecord, HistoryTransactionType, PoolLimits, TxType,
+         TransferConfig, TransferRequest, TreeState, ProverMode, HistoryRecordState,
+        } from 'zkbob-client-js';
+import { deriveSpendingKeyZkBob, bufToHex, nodeToHex, hexToBuf } from 'zkbob-client-js/lib/utils';
 import qrcodegen from "@ribpay/qr-code-generator";
 import { toSvgString } from "@ribpay/qr-code-generator/utils";
 import JSZip from "jszip";
-import assert from 'assert';
+import { env } from './environment';
 var pjson = require('../package.json');
 
 const bs58 = require('bs58');
-import { deriveSpendingKeyZkBob } from 'zkbob-client-js/lib/utils';
 
 
+export async function currentPoolEnvironment() {
+    const curPool = await this.account.getCurrentPool();
+    const poolEnv = env.pools[curPool];
+    const chainEnv = env.chains[String(poolEnv.chainId)];
+
+    this.echo(`Current pool: ${curPool}`);
+    this.echo(`Chain:        ${this.account.networkName()} (${poolEnv.chainId})`)
+    this.echo(`Pool address:     [[!;;;;${this.account.getAddressUrl(poolEnv.poolAddress)}]${poolEnv.poolAddress}]`);
+    this.echo(`Token address:    [[!;;;;${this.account.getAddressUrl(poolEnv.tokenAddress)}]${poolEnv.tokenAddress}]`);
+    this.echo(`RPC endpoint${chainEnv.rpcUrls.length > 1 ? 's' : ''}:     ${chainEnv.rpcUrls.join(', ')}`);
+    this.echo(`Relayer${poolEnv.relayerUrls.length > 1 ? 's' : ''}:          ${poolEnv.relayerUrls.join(', ')}`);
+    this.echo(`Cold storage:     ${poolEnv.coldStorageConfigPath}`);
+    this.echo(`Delegated prover${poolEnv.delegatedProverUrls.length > 1 ? 's' : ''}: ${poolEnv.delegatedProverUrls.join(', ')}`);
+    this.echo(`Minter:           ${env.minters[curPool]}`);
+    this.echo(`Cloud API:        ${env.cloudApi[curPool]}`);
+    this.echo(`UI URL:           ${env.redemptionUrls[curPool]}`);
+}
+
+export async function getAvailablePools() {
+    const pools: string[] = await this.account.getPools();
+    this.echo(`Available pools: ${pools.join(', ')}`);
+}
+
+export async function switchPool(poolAlias: string, password: string) {
+    if (!poolAlias) {
+        const pools: string[] = await this.account.getPools();
+        this.echo(`[[;red;]Please provide a pool alias. Currently supported: ${pools.join(', ')}]`)
+        return;
+    }
+    if (!password) {
+        this.echo(`[[;red;]Please provide account password to switch L1 account]`)
+    }
+
+
+    this.pause();
+    await this.account.switchPool(poolAlias, password);
+    this.resume();
+    this.echo(`Current pool: ${await this.account.getCurrentPool()}`);
+}
 
 export async function setSeed(seed: string, password: string) {
-    await this.account.login(seed, password);
+    //await this.account.login(seed, password);
```

Comment:
I guess we can just remove the whole function.

Reply:
Removed in https://github.com/zkBob/zkbob-console/pull/63/commits/039e8a8a86e788daa8464cb657922321045dcb70